### PR TITLE
The default configuration breaks projects that use SemanticUI.

### PR DESCRIPTION
**Pull request by Lemmih**
This patch explains why it happens and how to get around it.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -176,6 +176,16 @@ If you get an error in a production build, you can use sourcemaps to decode stac
 
 See [#61](https://github.com/stereobooster/react-snap/pull/61)
 
+### Semantic UI
+
+[Semantic UI](https://semantic-ui.com/) is defined over class substrings that contain spaces
```
Comment (stereobooster) [inline Readme.md]: 8047b7733dc407528457003089429dc6d26306ce6d6372c67f55caf8170fb003

**Comment by stereobooster** [comment]
Maybe I should disable class name sorting by default, but I do not like to have this in documentation. They use different dirty tricks in Semantic UI - I do not want to list them all in Readme.

- https://github.com/stereobooster/react-snap/issues/64
- https://github.com/stereobooster/react-snap/issues/27

**Comment by Lemmih** [comment]
You don't want to have a section about SemanticUI in the readme?

**Comment by stereobooster** [comment]
Never mind. I will reorganize Readme later. Thanks for contribution. Everybody who uses SemanticUI would be grateful for it.

**Comment by stereobooster** [comment]
> You can create a more sustainable community just by being nice. So don’t review PRs before you’ve had your morning coffee! ☕️ Take care of yourself so you can take care of your contributors.
> –@eileencodes 

Yeah this was before my coffee. Don't take it personal, please

**Pull request merged**